### PR TITLE
Feature/volume optimizations fe

### PR DESCRIPTION
**Pull request by volumio**

### CODE REVIEW

**Comment by volumio** [comment]
@kurtommy can you comment on this PR? Is it mergeable ? Do you spot any potential issue?

File: src/app/services/player.service.js [hunk 1]
```diff
@@ -191,6 +194,23 @@ class PlayerService {
     this.socketService.emit('volume', volume);
   }
 
+  get sliderVolume(){
+    // console.log("slider get");
+    if(Date.now() - this.volumeSliderLastUpdateTime > 1000){
```
Comment (kurtommy) [inline src/app/services/player.service.js]: It's better to expose to the UI the prop `volume` and not to bind the var name to the UI component that will consume that variable.
So rename the vars to something similar to:
volumeSliderLastUpdateTime -> lastVolumeUpdateTime
volumeSliderValue -> volume
this.volume -> lastVolumeValue

File: src/app/themes/volumio/components/volume-manager/volumio-volume-manager.html [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
 <div ng-if="volumeManager.matchmediaService.isPhone" class="volumeManager">
   <div class="volumeSliderWrapper" ng-show="volumeManager.showVerticalSlider">
-    <div class="bigText">{{volumeManager.playerService.volume}}</div>
```
Comment (kurtommy) [inline src/app/themes/volumio/components/volume-manager/volumio-volume-manager.html]: keep .volume

Comment (crisp00) [reply src/app/themes/volumio/components/volume-manager/volumio-volume-manager.html]: The text is meant to be an indicator of the slider's value. Isn't it more logically correct if the text mirrors the direct value of the slider, which in term mirrors the volume of the state, rather then them both mirroring the state?

From a functional perspective, the problem is that the text would only update with the slider's value after the BE has reacted to the change, which means lagg.

**Pull request merged**